students/mark/Session11/create_danceable.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+"""
+Session 11:
+
+Your job, now, is to:
+
+* get artists
+* get song names for for tracks
+with:
+* danceability scores over 0.8
+* and loudness scores below -5.0.
+
+In other words, quiet yet danceable tracks.
+
+Also, these
+
+* tracks should be sorted in descending order by danceability
+    so that the
+* most danceable tracks are up top.
+
+You should be able to
+
+* work your way there starting with the comprehension above.
+
+And while you could use Pandas features along the way, you don’t need to.
+To accomplish the objective you do not need to know anything more about Pandas than what
+you can infer from the material herein. Standard library functions that could come in handy
+include
+
+* can use: zip() and sorted().
+
+Submit your code and the top five tracks to complete the assignment.
+"""
+
+
+import pandas as pd
+music = pd.read_csv("./data/featuresdf.csv")
+
+### Take a look around to get a sense of the general shape of the data.
+#### Uneeded after debugging (remove from production code / here only as example of looking around)
+# print(music.head())
+# print('#' * 3)
+# music.describe()
+
+### Now we are ready for the analytics. This first one is a gimme.
+### We will use a comprehension to get danceability scores over 0.8.
+# print([x for x in music.danceability if x > 0.8])
+# print([x for x in music.loudness if x < -5.0])
+# print([x for (x,y) in zip(music.loudness, music.danceability) if x < -5.0])
+# print([(x,y) for (x,y) in zip(music.loudness, music.danceability) if x < -5.0 and y > 0.8])
+
+def create_sexy_music_list():
+
+    return [(z,n,x,y) for (n,x,y,z) in zip(music.name, music.artists, music.loudness, music.danceability) if y < -5.0 and z > 0.8]
```

Comment:
try splitting these into multiple lines of code. I like how you made this a function though. However, if it were really one line of code, then there's no need to really break into a function unless you were calling it all the time.